test/utils.go

```diff
@@ -66,6 +67,14 @@ func getBoskosProject(resourceType string) *common.Resource {
 	return project
 }
 
+func releaseBoskoProject(project string) error {
+	err := boskos.ReleaseOne(project, "free")
+	if err != nil {
+		return fmt.Errorf("bosko failed to release project: %w", err)
```

Comment:
if you are going to wrap errors, the standard thing to do is to add the context around the call:

fmt.Errorf("releaseBoskoProject(%s): %w", project, err)

otherwise you are not adding any more information to the error, just adding redundant text.

Reply:
Is this called from somewhere else? Do we need a separate func for this?

Reply:
It is calling from setupProwConfig().

Reply:
That's what I'm pointing out -- it probably should just be inlined.